### DISCUSSION TITLE
can i use remote-dom with angular ?

### DISCUSSION THREAD

**Discussion by SavanAghera** [Shopify/remote-ui#379] [Q&A]
if yes how can i !
 I am exploring stuff 
 is there any alternatives ?

**Comment by lemonmade** [comment]
Hi @SavanAghera. I have not actually tried to use Angular on top of Remote DOM, but I don't know of anything Angular does with the DOM that would make it incompatible. Other frameworks like [Vue, React, and Svelte](https://github.com/Shopify/remote-dom/tree/main/examples/kitchen-sink) all work fine. That said, I wasn't able to easily find a way to run a very simple Angular app in isolation; Angular has a lot of opinions about the build process and the "HTML entrypoint" that I am not expert enough to sort out. In contrast, these other libraries generally let you create and render a component easily, as long as you have a DOM node (which is what Remote DOM will provide you with); for example, Svelte [lets you import a Svelte component and render it directly to an HTML element](https://github.com/Shopify/remote-dom/blob/main/examples/kitchen-sink/app/remote/examples/svelte.ts#L7).
